pom.xml

```
@@ -105,6 +105,24 @@
             <artifactId>spring-boot-starter-security</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.jsonwebtoken</groupId>
+            <artifactId>jjwt-api</artifactId>
+            <version>0.11.5</version>
```

same comment for other places
```suggestion
            <version>${jjwt.version}</version>
```